static_rdf_server/app.py

```diff
@@ -0,0 +1,48 @@
+"""Module for server."""
+import logging
+import os
+
+from aiohttp import web
+from aiohttp_middlewares import cors_middleware, error_middleware
+from dotenv import load_dotenv
+
+load_dotenv()
+LOGGING_LEVEL = os.getenv("LOGGING_LEVEL", "INFO")
+HELLO_WORLD = '<http://example.com/server> <http://example.com/says> "Hello World" .'
+
+
+async def get_ontology(request: web.Request) -> web.Response:
+    """Return default response."""
+    folder = request.match_info["folder"]
+    ontology = request.match_info["ontology"]
+    logging.debug(f"Got request for folder/file {folder}/{ontology}")
+
+    if "examples" == folder:
+        return web.Response(text=HELLO_WORLD, content_type="text/turtle")
+    else:
+        full_path = os.path.join(os.sep, "var", folder, ontology + ".ttl")
```

Comment:
Bruk helst ikke /var men for eks /var/static og gi kun lese tilgang i containeren.

Reply:
Enig, har leste meg fram til at det beste er å bruke `/srv/www/`, feks `srv/www/static-rdf-server/<folder>/<ontolgy>/<ontology.ttl>`, ref nederst i ny versjon av README

Reply:
/var/www er vanlig brukt til web content for eks apache bruker dette by default. Ikke så viktig egentlig men det som er viktig er at folderen ikke har skrivetilgang.